src/Sql/Sql/help/Disable-AzSqlDatabaseLedgerDigestUpload.md

```diff
@@ -37,12 +37,12 @@ The Disable-AzSqlDatabaseLedgerDigestUpload cmdlet disables uploading ledger dig
 
 ### Example 1
 ```powershell
-PS C:\> Disable-AzSqlDatabaseLedgerDigestUpload -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -DatabaseName "Database01" 
-```
+PS C:\> Disable-AzSqlDatabaseLedgerDigestUpload -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -DatabaseName "Database01"
 
 ResourceGroupName ServerName DatabaseName State    Endpoint
 ----------------- ---------- ------------ -----    --------
 ResourceGroup01   Server01   Database01   Disabled
+```
```

Comment:
there needs a description. Please check your examples.

Reply:
I agree, but I'm not familiar with these cmdlets, and it's a out of the scope of this PR. These help files got modified by running platyPS (that's why I didn't initially revert the changes)